src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UITests/Presentation/SampleChooserViewModel.cs

```diff
@@ -797,11 +826,14 @@ void Dispose(object snd, RoutedEventArgs e)
 					recents.RemoveAt(_numberOfRecentSamplesVisible);
 				}
 
+				Console.WriteLine("UpdateContent SetFile");
 				await SetFile(SampleChooserLRUConstant, recents.ToArray());
 
 				RecentSamples = recents;
 			}
 
+			Console.WriteLine("UpdateContent Done");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6226524)